### PR TITLE
Add boardgame provider

### PR DESCRIPTION
**Pull request by jensGiehl**
Data taken from boardgamegeek.com and recommend.games

### CODE REVIEW

File: src/test/java/net/datafaker/providers/base/BoardgameTest.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package net.datafaker.providers.base;
+
+import java.util.Collection;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (kingthorin) [inline src/test/java/net/datafaker/providers/base/BoardgameTest.java:6]: Please use specific imports

**Review by kingthorin — COMMENTED** [review]
This should probably be an Entertainment provider

File: src/test/java/net/datafaker/providers/base/BoardgameTest.java (hunk 1 above)
Comment (jensGiehl) [reply src/test/java/net/datafaker/providers/base/BoardgameTest.java:6]: The import is even completely superfluous. Thanks for the tip!

**Comment by jensGiehl** [comment]
> This should probably be an Entertainment provider

For Entertainment, I wasn't sure if it belonged there, since only movies are listed there so far.

File: src/main/java/net/datafaker/providers/base/Boardgame.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
+package net.datafaker.providers.base;
+
+/**
+ * since 2.2.3
```
Comment (snuyanzin) [inline src/main/java/net/datafaker/providers/base/Boardgame.java:4]: why is it not an annotation?

Comment (jensGiehl) [reply src/main/java/net/datafaker/providers/base/Boardgame.java]: fixed :)

**Comment by bodiam** [comment]
Hi @jensGiehl , thanks for this contribution! Looks great, and thanks for addressing the review comments!

**Comment by kingthorin** [comment]
Geez you were quick. I read the comment about Entertainment and went and look, I almost told you that you merged to quickly and that there was already a Boardgame provider, then I realized the file I was looking at was only 2 minutes old 🤣

**Pull request merged**